protocol/types_gen.go

```diff
@@ -1307,7 +1307,7 @@ func (v *SigSpecGroup) MarshalJSON() ([]byte, error) {
 	}
 	u.ChainHeader = v.ChainHeader
 	u.SigSpecs = encoding.ChainSetToJSON(v.SigSpecs)
-	return json.Marshal(u)
+	return json.Marshal(&u)
```

Comment:
Changes need to be made to `./internal/cmd/gentypes` - generated files should not be manually edited. CI is failing because this change desyncrhonizes `types.yml` and `types_gen.go`.